t/03components.t

```diff
@@ -83,6 +83,10 @@ use DateTime;
     ok( !$d->is_leap_year,         '->is_leap_year' );
     ok( !$d->is_last_day_of_month, '->is_last_day_of_month' );
 
+    is ( $d->month_length,   31,  '->month_length' );
+    is ( $d->quarter_length, 92,  '->quarter_length' );
+    is ( $d->year_length,    365, '->year_length' );
+
```

Comment:
This needs tests for leap years too, and ideally for every quarter in both leap & non-leap years.